Code/app/src/main/java/com/cmpt362/fitcheck/firebase/User.kt

```diff
@@ -0,0 +1,3 @@
+package com.cmpt362.fitcheck.firebase
+
+data class User(var email: String? = "", var uid: String? = "") {}
```

Comment:
Should we make  email and uid nullable, since we will always have an email and uid for every user?